### DISCUSSION TITLE
选择后的效果，怎么在其他地方取消

### DISCUSSION THREAD

**Discussion by xllyll** [zouyaoji/vue-cesium#533] [General]
![微信截图_20240320102501](https://github.com/zouyaoji/vue-cesium/assets/7789193/6809236d-7ce1-488a-b057-e59b7f16b49c)
就是如何再其他的按钮点击后取消绿色的选中框

**Comment by zouyaoji** [comment]
可以设置 viewer.selectedEntity 来取消
